src/components/Receiving/ItemDetails.js

```diff
@@ -6,62 +6,148 @@ import { get } from 'lodash';
 
 import {
   Button,
+  Checkbox,
+  KeyValue,
   Modal,
+  MultiColumnList,
   Row,
   Select,
   TextField,
-  MultiColumnList,
 } from '@folio/stripes/components';
 
+import css from './ItemDetails.css';
+
+const ITEM_STATUS = {
+  received: 'Received',
+};
+
 class ItemDetails extends Component {
   static propTypes = {
     close: PropTypes.func.isRequired,
     itemList: PropTypes.arrayOf(PropTypes.object).isRequired,
+    locationsOptions: PropTypes.arrayOf(PropTypes.object).isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      allChecked: false,
+      checkedItems: [],
+    };
+  }
+
+  getFooter = (close) => (
+    <Row end="xs">
+      <Button
+        onClick={close}
+      >
+        <FormattedMessage id="ui-orders.receiving.cancelBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled
+      >
+        <FormattedMessage id="ui-orders.receiving.previousBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled={!this.state.checkedItems.length}
+      >
+        <FormattedMessage id="ui-orders.receiving.nextBtn" />
+      </Button>
+    </Row>
+  );
+
+  toggleAll = () => {
+    const { itemList } = this.props;
+
+    this.setState(({ allChecked }) => ({
```

Comment:
I would add more code for better understanding,
```
this.setState((state, props) => {
  const allChecked = !state.allChecked;
  const checkedItems = allChecked ? props.itemList || [];
  return {
    allChecked,
    checkedItems,
  };
});
```

Reply:
IMO, it's more visible than `checkedItems: allChecked ? [] : itemList,` - where you set empty array if `allChecked` is truthy